### DISCUSSION TITLE
Recommended best practices to handle scoping of tailwindcss in a microfrontend setup.

### DISCUSSION THREAD

**Discussion by Bjodol** [tailwindlabs/tailwindcss#9229] [Help]
If you are using a microfrontend setup that does run time compisition of apps (like single spa) from several remotes what would be the best practices to include tailwind?

I thought about the important selectore: `important:"#microApp"`, but based on the documentation it only applies to utilities. In cases where you use design system plugins that adds components you would also like to avoid those from leaking between apps. 

Do you have any recommendations for how to set up remote microfrontends that also uses tailwind components?

**Comment by olets** [comment]
Have you looked at the `prefix` config option? https://tailwindcss.com/docs/configuration#prefix
